cmd/entrypoint/main.go

```diff
@@ -181,6 +183,14 @@ func main() {
 		case termination.MessageLengthError:
 			log.Print(err.Error())
 			os.Exit(1)
+		case entrypoint.ContextError:
+			if errors.Is(err, entrypoint.ErrContextCanceled) {
+				log.Print("Step was cancelled")
+				os.Exit(int(syscall.SIGKILL))
```

Comment:
curious why SIGKILL is used here? could you add a comment about this?

Reply:
use the SIGKILL signal to distinguish normal exit programs, just like kill -9 PID.

added comment